scripts/references/create_allowed_species.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python3
+
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+""" Script to create the reference allowed_species.json automatically form the reference mlss_conf.xml """
+
+
+import sys
+import json
+import argparse
+import xml.etree.ElementTree
+from xml.etree.ElementTree import ParseError
+from typing import List
+
+
+def get_species_list(mlss_conf_file: str) -> List:
+    """
+    Return a list with all species present in the mlss_conf.xml. Each species are unci in the list
+
+    :param mlss_conf_file: path to the mlss_conf.xml
+    :return: list of species
+    """
+
+    try:
+        compara_db_tree = xml.etree.ElementTree.parse(mlss_conf_file)
+    except ParseError as err:
+        print("issue with " + mlss_conf_file)
+        print(err)
+        sys.exit(1)
+
+    root = compara_db_tree.getroot()
+    genomes = root.findall("./collections/collection/genome")
+    species_dic = {}
+    for genome in genomes:
+        species_dic[genome.get('name')] = 1
+    species_list = list(species_dic.keys())
+    return species_list
```

Comment:
I'm wondering if we should `sort` the list of species. This way, the number of differences between runs/commits will be much smaller and easier to track.